t/ffi/lpcwstr.c

```diff
@@ -0,0 +1,76 @@
+#include "libtest.h"
+#include <stdint.h>
+#include <stdlib.h>
+#include <string.h>
+
+typedef uint16_t     WCHAR;
```

Comment:
I think we can just use the `wchar_t*` convention here instead of introducing the Windows aliases.  File can be called widestr.c